jbpm-wb-process-instance-migration/pim-react/src/component/MainPagewithPfTab.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import classNames from "classnames";
+import axios from "axios";
+
+import {
+  TabContainer,
+  Nav,
+  NavItem,
+  TabPane,
+  TabContent
+} from "patternfly-react";
+import { DropdownButton, MenuItem } from "patternfly-react";
+
+import MigrationPlans from "./tabMigrationPlan/MigrationPlans";
+import MigrationDefinitions from "./tabMigration/MigrationDefinitions";
+import { MockupData_KIE_SERVER_ID } from "./common/MockupData";
+import { USE_MOCK_DATA } from "./common/PimConstants";
+import { BACKEND_URL } from "./common/PimConstants";
+
+export default class MainPagewithPfTab extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      kieServerIds: "",
+      title: "KIE Server Name",
+      menuItems: []
+    };
+  }
+
+  componentDidMount() {
+    let kieServerIds;
+    let menuItems = this.state.menuItems;
+    if (USE_MOCK_DATA) {
```

Comment:
U mean remove the USE_MOCK_DATA? I don't have such plan. 
This option is easy for UXD team to test the frond-end without setup the whole BC, KIE, PIM-services. 
For example, Sarah can work independently using this mode to do CSS upgrade, etc.

Reply:
I understand the value for demonstration and other purposes.

As a (future?) improvement I'd propose a mocked pim service serving the mock data. The service would be again used only when a certain profile is activated.

Reply:
I'm not sure using a mock service (which involves backend java code?) is a better idea. Because right now the MOCK_DATA is served inside React, there is no REST involved, and end-user don't need to start any JVM, all they need is node server.